app/lib/app/SecvisogramPage/View/FormEditor/editors/GenericEditor/Attributes/CVSS3Attribute.js

```diff
@@ -94,153 +93,203 @@ export default function CVSSV3Attribute({
-          disabled={false}
+        <Collapsible startCollapsed={true} title="base inputs">
+          {dropdownFor('attackVector', [
+            'NETWORK',
+            'ADJACENT_NETWORK',
+            'LOCAL',
+            'PHYSICAL',
+          ])}
+          {dropdownFor('attackComplexity', ['HIGH', 'LOW'])}
+          {dropdownFor('privilegesRequired', ['NONE', 'HIGH', 'LOW'])}
+          {dropdownFor('userInteraction', ['NONE', 'REQUIRED'])}
+          {dropdownFor('scope', ['UNCHANGED', 'CHANGED'])}
+          {dropdownFor('confidentialityImpact', ['NONE', 'HIGH', 'LOW'])}
+          {dropdownFor('integrityImpact', ['NONE', 'HIGH', 'LOW'])}
+          {dropdownFor('availabilityImpact', ['NONE', 'HIGH', 'LOW'])}
+        </Collapsible>
+        <div
+          className={`p-2 rounded border ${getSeverityColors(
+            /** @type {number} */ (value?.baseScore)
+          )}`}
         >
-          {typeof value?.baseScore === 'number' ? String(value.baseScore) : ''}
-        </Attribute>
-        <Attribute
-          label={'BaseSeverity'}
-          description={'The CVSS Base Severity'}
-          instancePath={instancePath.concat(['baseSeverity'])}
-          property={property}
-          disabled={false}
+          <Attribute
+            label={'BaseScore'}
+            description={'The CVSS Base Score'}
+            instancePath={instancePath.concat(['baseScore'])}
+            property={property}
+            disabled={false}
+          >
+            {typeof value?.baseScore === 'number'
+              ? String(value.baseScore)
+              : ''}
+          </Attribute>
+        </div>
+        <div
+          className={`p-2 rounded border ${getSeverityColors(
+            /** @type {number} */ (value?.baseScore)
+          )}`}
         >
-          {value?.baseSeverity || ''}
-        </Attribute>
-        {dropdownFor(
-          'exploitCodeMaturity',
-          ['UNPROVEN', 'PROOF_OF_CONCEPT', 'FUNCTIONAL', 'HIGH', 'NOT_DEFINED'],
-          false
-        )}
-        {dropdownFor(
-          'remediationLevel',
-          [
-            'OFFICIAL_FIX',
-            'TEMPORARY_FIX',
-            'WORKAROUND',
-            'UNAVAILABLE',
-            'NOT_DEFINED',
-          ],
-          false
-        )}
-        {dropdownFor(
-          'reportConfidence',
-          ['UNKNOWN', 'REASONABLE', 'CONFIRMED', 'NOT_DEFINED'],
-          false
-        )}
-        <Attribute
-          label={'TemporalScore'}
-          description={'The CVSS Temporal Score'}
-          instancePath={instancePath.concat(['temporalScore'])}
-          property={property}
-          disabled={false}
+          <Attribute
+            label={'BaseSeverity'}
+            description={'The CVSS Base Severity'}
+            instancePath={instancePath.concat(['baseSeverity'])}
+            property={property}
+            disabled={false}
+          >
+            {value?.baseSeverity || ''}
+          </Attribute>
+        </div>
```

Comment:
The Socres and Severity should be above the `Collapsible` not below. (Applies to all instances in this PR)

e.g. Move 111-142 between 95 and 96

Reply:
changed